.ci/actions/os-preparation/matrix.json

```diff
@@ -0,0 +1,16 @@
+[
+  {
+    "os": ["ubuntu-latest"],
+    "event":"pull_request"
+  }, 
+  {
```

Comment:
sorry for being picky here, but could we please follow the same order for all entries (os, event, filter) ?
```
{
    "os": ["ubuntu-latest", "windows-latest"],
    "event":"pull_request",
    "repository": ["drools", "droolsjbpm-knowledge", "kie-jpmml-integration", "openshift-drools-hacep", "optaplanner"]
  }
```

Reply:
you are right, just did!